serde/src/lib.rs

```diff
@@ -91,14 +91,22 @@ pub trait SerializedMaxSize {
     const MAX_SIZE: usize;
 }
 
+/// Types implementing this trait have an alternative fixed-size serialization length.
+pub trait SerializedFixedSize {
```

Comment:
I think a better naming could be `FixedSerializedSize`. wdyt?

Reply:
I think I prefer the current name.